### PR TITLE
feat(forms): add Upload field

### PR DESCRIPTION
**Pull request by tujoworker**

### CODE REVIEW

**Review by langz — APPROVED** [review]
Looks great 👏 

I tested the deploy preview, and have a few observations to report from testing the [required example](https://eufemia-git-feat-upload-field-eufemia.vercel.app/uilib/extensions/forms/feature-fields/Upload/#required):

- I'm able to submit the Form when providing a too big/large file, see video 1.
- I'm able to submit the Form without having any selected files(when first selecting a file, then removing/deleting the same file), see video 2.
- Just a general question, should I be able to submit a form containing one too big file and a valid file? Or should it result in an error? see video 3

Video 1:


https://github.com/dnbexperience/eufemia/assets/1359205/9032e48c-36fa-4e18-80c0-ccc61a532605



Video 2:





https://github.com/dnbexperience/eufemia/assets/1359205/a21cc6bb-8be0-4e05-94fd-0ce71e037ec2




Video 3:

https://github.com/dnbexperience/eufemia/assets/1359205/21e9e6e0-19cd-4b94-b6e6-ff0476ab96a6

File: packages/dnb-eufemia/src/components/upload/UploadDocs.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { PropertiesTableProps } from '../../shared/types'
+
+export const UploadProperties: PropertiesTableProps = {
+  acceptedFileTypes: {
+    doc: 'List of accepted file types. ',
```
Comment (langz) [inline packages/dnb-eufemia/src/components/upload/UploadDocs.ts:5]:
```suggestion
    doc: 'List of accepted file types.',
```

**Comment by tujoworker** [comment]
Thank you for the good feedback. Yes, the user should resolve the error (remove the file) to be able to submit. This is in align to how all other "fields" needs to be handled by the user.

**Pull request merged**